dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilder.java

```diff
@@ -68,15 +69,20 @@ public ClassPathResult resolve(List<Artifact> artifacts) throws RepositoryExcept
     }
     // dependencyGraph holds multiple versions for one artifact key (groupId:artifactId)
     DependencyGraphResult result =
-        dependencyGraphBuilder.getStaticLinkageCheckDependencyGraph(artifacts);
+        dependencyGraphBuilder.buildLinkageCheckDependencyGraph(artifacts);
```

Comment:
Renamed getStaticLinkageCheckDependencyGraph to build ... to follow the previous refactoring.

Reply:
Good idea, though it's a little easier if this sort of change can happen in its own PR that takes less time to review and approve